src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,240 @@
+and the resolvable version of `pedantic` is **1.9.0**.
+
+{{site.alert.tip}}
+  To see what changed in the new version of a package
+  that's published on [pub.dev,]({{site.pub}})
+  look at the changelog in the package page.
+  For example, you can look at the **Changelog** tabs on the pages for the
+  [`args`][] and [`pedantic`][] packages.
+{{site.alert.end}}
+
+To fix the first dependency (`args`),
+all that's needed is to run `pub upgrade`:
+
+```terminal
+$ pub upgrade
+Resolving dependencies... 
+> args 1.6.0 (was 1.5.0)
+  pedantic 1.8.0 (1.9.0 available)
+Changed 1 dependency!
+```
+
+To fix the second dependency (`pedantic`),
+you can change the pubspec's `pedantic` entry to use [caret syntax][] (`^`)
+and the number in the **Resolvable** column: **`^1.9.0`**.
+Here's the diff for `pubspec.yaml`:
+
+{% comment %}
+  [TODO: Improve the formatting of the following diff]
+{% endcomment %}
+
+```diff
+-  pedantic: 1.8.0
++  pedantic: ^1.9.0
+```
+
+{{site.alert.note}}
+  Updating the `pedantic` package can cause code to start failing analysis,
+  depending on your code and how you include the analysis options file.
+  For details, see the section on
+  [enabling default Google rules][using pedantic]
+  in [Customizing static analysis][].
+{{site.alert.end}}
+
+After editing `pubspec.yaml`, run `pub upgrade` to
+update the `pubspec.lock` file.
+You can then run `pub outdated` to confirm that
+you've made all necessary changes.
+
+```terminal
+$ pub upgrade
+...
+$ pub outdated
+Found no outdated packages.
+```
+
+{{site.alert.important}}
+  Test your code to verify that it still works as expected
+  after updating the packages.
+{{site.alert.end}}
+
+
+## Output columns
+
+The output of `pub outdated` has four columns of version information
+for each out-of-date dependency:
+
+```terminal
+$ pub outdated
+Dependencies  Current  Upgradable  Resolvable  Latest  
+args          1.5.0    1.6.0       1.6.0       1.6.0   
+
+dev_dependencies
+pedantic      1.8.0    1.8.0       1.9.0       1.9.0   
+
+transitive dependencies: all up-to-date
+
+transitive dev_dependencies: all up-to-date
+
+1 upgradable dependency is locked (in pubspec.lock) to an older version.
+To update it, use `pub upgrade`.
+
+1 dependency is constrained to a version that is older than a resolvable version.
+To update it, edit pubspec.yaml.
+```
+
+Here are the meanings of each column:
+
+Current
+: The version used in your package, as recorded in `pubspec.lock`.
+  If the package isn't in `pubspec.lock`,
+  the value is `-`.
+
+Upgradable
+: The latest version allowed by your `pubspec.yaml` file.
+  This is the version that `pub upgrade` resolves to.
+  The value is `-` if the value in the **Current** column is `-`.
+
+Resolvable
+: The latest version that can be resolved,
+  when combined with all other dependencies.
+  This version corresponds to what you get if
+  you set all version constraints in `pubspec.yaml` to `any`
```

Comment:
"you set" sounds a little too much like a recommendation for something to do. Maybe phrase like "corresponds to what you get if you had an `any` version constraints in `pubspec.yaml`  on all dependencies" ?